src/utility/helpers.unit.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This moved down into src/ so we could make that directory the single common parent of all import-able things.

Reply:
Sounds good. Maybe a `test` folder instead? Something to distinguish from the other things in `utilities` which are actually app-related behavior.

Also, please make sure this file is ignored by Jest for code coverage checks. (`package.json` and `jest.config.json`).

Reply:
Opted for a top-level `testhelpers` folder so the `utilities` folder stays completely for app code. Ignored in both places 👍